Readme.md

```diff
@@ -1,63 +1,66 @@
-
-
-```xaml
-<dxdo:DocumentPanel x:Name="docPanel2" Caption="Panel 2" Content="{dxdo:RelativeUri UriString=CustomWindows\\MyWindow.xaml}"/>
-```
-
-
-<p><br>2) The DocumentPanel.Content property is set with a Uri object at runtime:</p>
-
-
-```cs
-docPanel1.Content = new Uri(@"CustomWindows\MyWindow1.xaml", UriKind.Relative);
-```
-
-
-<p> </p>
-<p>3) The <a href="https://documentation.devexpress.com/#WPF/DevExpressXpfDockingDockLayoutManager_DockControllertopic">DockLayoutManager.DockController</a>.<a href="https://documentation.devexpress.com/#WPF/DevExpressXpfDockingDockControllerBase_AddDocumentPaneltopic">AddDocumentPanel</a> method creates a new DocumentPanel object and loads the contents of an external XAML file into the created panel. </p>
-
-
-```cs
-panel1 = dockLayoutManager1.DockController.AddDocumentPanel(documentGroup1,
-
-    new Uri(@"CustomWindows\UserControl1.xaml", UriKind.Relative));
-
-panel1.Caption = "Document " + (ctr++).ToString();
-```
-
-
-<br>
-<p>In the example, the XAML file defines a UserControl object. The loaded UserControl is accessed via the DocumentPanel's Control property and then a method on the UserControl is invoked.</p>
-
-
-```cs
-//...
-
-(panel1.Control as UserControl1).SetDataContext(imageInfo);
-```
-
-
-<br>
-<p>You can see this in action by clicking the "Set DataContext for UserControl" button in the example.</p>
-
-<br/>
-
-
+<!-- default badges list -->
+[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E2410)
+[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
+<!-- default badges end -->
+
+# WPF Dock Layout Manager - Load an External Window or UserControl into a DocumentPanel
+
+You can define a Window, Page, or UserControl in external XAML files and load their contents into [DocumentPanel](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager.DocumentPanel) objects.
+
+This example loads an external Window and UserControl into DocumentPanels in three ways:
+
+1. Use the [DocumentPanel.Content](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.ContentItem.Content) inherited property to load the content of _MyWindow.xaml_ into a [DocumentPanel](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager.DocumentPanel) at design time (in XAML). The **Content** property accepts an [Uri](https://docs.microsoft.com/en-us/dotnet/api/system.uri) object that is linked to a XAML file and defines a Window, Page, or UserControl.

```

Comment:
Note that the article before "Uri" should be "a" (not "an"). Can you please update this line?